README.md

```diff
@@ -11,9 +11,9 @@ Opening blurb (Brian to come up with something great....)
  * [Kntyre Speedtest Add-on for Splunk](./kintyre-speedtest-TA) - Speedtest data collection for
    splunk (universal forwarder) endpoints
 
-The Kintyre Speedtest Technology Add on uses the standalone Shinnecock Agent to capture and collect
+The Kintyre Speedtest Add-on uses the standalone Shinnecock Agent to capture and collect
```

Comment:
Let's keep "Technology" here.   It's a bit legacy, but it does help explain what "TA" stands for, even though the "Technology" bit is often dropped other places (like above and in the Splunk UI)

"Add-on" is correct, good eye.

Reply:
Besides. I need something to "complain" about so I can use github's cool "review" features ;-)